email-templates/weekly/html.ejs

```diff
@@ -0,0 +1,579 @@
+                                      </table>
+                                    </div>
+                                  </td>
+                                </tr>
+                              </table>
+                            </a>
+                          </td>
+                        </tr>
+                        <tr>
+                          <td class="preview-asset-metadata">
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_thumbs_up.png') %>" alt="Number of likes" /> <%= item.asset.likes %>
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_views.png') %>" alt="Number of views" /> <%= item.asset.views %>
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_comments.png') %>" alt="Number of comments" /> <%= item.asset.comment_count %>
+                          </td>
+                        </tr>
+                        <tr>
+                          <td class="centered">
+                            <div class="outbound-button">
+                              <a href="<%= assetData.link %>" target="_blank">
+                                Check it out
+                              </a>
+                            </div>
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                <%
+                });
+                -%>
+                </tr>
+              </tbody>
+            </table>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <table cellpadding="0" cellspacing="5" class="graph-container">
+              <tbody>
+                <%
+                var mostGenerated = weekly.course.topUsers.pointsGenerated;
+                var mostReceived = weekly.course.topUsers.pointsReceived;
+
+                var maximumPoints = _.chain([mostGenerated, mostReceived]).map('total').max().value();
+                var scaleFactor = 200 / maximumPoints;
+                -%>
+
+                <tr>
+                  <td class="most-points-outer-left">
+                  <%
+                  if (mostGenerated) {
+                  -%>
+                    <h4 class="section-header">
+                      Most points generated
+                    </h4>
+                  <% 
+                  } 
+                  -%>
+                  </td>
+                  <td class="most-points-outer-right">
+                  <%
+                  if (mostReceived) {
+                  -%>
+                    <h4 class="section-header">
+                      Most points received
+                    </h4>
+                  <% 
+                  } 
+                  -%>
+                  </td>
+                </tr>
+
+                <tr>
+                  <td class="most-points-outer-left">
+                  <%
+
```

Comment:
@johncrossman Yes, the idea was to make this bit of control flow stand out without creating extra consecutive line breaks in the generated template. But if it looks weird we could do it another way - I haven't found an ejs scriplet style guide to date.